packages/terra-site/src/examples/section-header/SectionHeaderExampleTemplate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SectionHeader from 'terra-section-header';
+
+const propTypes = {
+  /**
+   * Title of the example
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * Props to be spread onto SectionHeader
+   */
+  /* eslint-disable react/forbid-prop-types */
```

Comment:
Should this be a next-line disable? Or should the rule be re-enabled?

Reply:
I suppose 

af0ecb6a6b6fb656bebc7680bc15bd295773c005